deploy/push-ecr-image.sh

```diff
@@ -13,9 +13,12 @@ AWS_ACCOUNT=$(aws sts get-caller-identity --query Account --output text)
 
 DOCKER_TAG="$(date +%Y%m%d)-$(git rev-parse --short=8 HEAD)"
 DOCKER_IMAGE=$AWS_ACCOUNT.dkr.ecr.$AWS_REGION.amazonaws.com/setup-app:$DOCKER_TAG
+DOCKER_IMAGE_LATEST=$AWS_ACCOUNT.dkr.ecr.$AWS_REGION.amazonaws.com/setup-app:latest
 
 aws ecr get-login-password --region "$AWS_REGION" | docker login --username AWS --password-stdin "$AWS_ACCOUNT".dkr.ecr."$AWS_REGION".amazonaws.com
-docker build -t  "$DOCKER_IMAGE" .
+aws ecr describe-repositories --repository-names setup-app --profile $AWS_PROFILE --region $AWS_REGION || aws ecr create-repository --repository-name setup-app --profile "$AWS_PROFILE" --region "$AWS_REGION"
```

Comment:
:thinking: -- This kind of conflicts (or at least muddies the water IMHO) with the "ownership" of creating repositories in the [IAC here](https://github.com/safeinsights/iac/tree/main/enclave-container-registry). Are we okay with that?

Reply:
In the TOA we have the repository creation, image build and push as part of the `npm scripts`, which I assume aims to easy build and deployment for devs, but we can bring it up tomorrow with the team and see how we want to go about it and how that can work with future CI-CD pipelines.

Reply:
did we want to circle back on this at all or are we good with these changes as is?

Reply:
:man_shrugging: -- I'm fairly indifferent. I think with the new roles / perms this may not work for devs anyways outside of the sandbox, though :sweat_smile: .